tests/ui/test_ui_tools.py

```diff
@@ -1513,10 +1515,12 @@ def edit_mode_view(self, mocker):
         (2, 'change_all'),
     ])
     @pytest.mark.parametrize('key', keys_for_command('ENTER'))
-    def test_select_edit_mode(self, mocker, edit_mode_view,
+    def test_select_edit_mode(self, mocker, edit_mode_view, widget_size,
                               index_in_widgets, mode, key):
         radio_button = edit_mode_view.widgets[index_in_widgets]
-        size = (20, 20)
+        size = widget_size(edit_mode_view)
```

Comment:
`edit_mode_view` or `radio_button`?

Reply:
This should have been `radio_button`. Thanks! Fixed for the other comments as well.